### PR TITLE
fiona / remove trailing slash on url

### PR DESCRIPTION
**Pull request by fiona-deriv**
Changes:

-   ...

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Comment by nikolaiberestevich-deriv** [comment]
LGTM :)

File: src/components/localization/localized-link.js [hunk 1]
```diff
@@ -164,7 +164,7 @@ const getURLFormat = (type, locale, to, affiliate_lang) => {
             type.replace(/_/g, '-') + '/'
         }`
     } else if (only_en_new_tab_no_modal.includes(type)) {
-        return `${localized_link_url.domain_full_url}/${type.replace(/_/g, '-') + '/'}`
```
Comment (habib-deriv) [inline src/components/localization/localized-link.js:167]: I wonder, if this would break other things or not

Comment (fiona-deriv) [reply src/components/localization/localized-link.js:167]: @habib-deriv,  prior to updating my master branch, the slash was missing but nothing seemed broken though
but, I'll try & test all other pages

Comment (nikolaiberestevich-deriv) [reply src/components/localization/localized-link.js:167]: It should't break. As I've tested. But Fiona's test will necessary too, I think

Comment (sean-binary) [reply src/components/localization/localized-link.js:167]: These are for pages outside of this project right?

**Review by sean-binary — APPROVED** [review]

**Pull request merged**